pkg/resourcemanager/controller/managedresource/reconciler.go

```diff
@@ -106,6 +106,11 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		return reconcile.Result{}, fmt.Errorf("could not fetch ManagedResource: %+v", err)
 	}
 
+	if ignoreMode(mr) && mr.DeletionTimestamp == nil {
+		log.Info("Skipping reconciliation due to ignore mode")
+		return reconcile.Result{}, nil
+	}
```

Comment:
Does it make sense to add envtest (make sure that the controller does not reconcile a ManagedResource excluded from reconciliation) or at least update the corresponding umbrella issue with missing integration test cases for this scenario (ref https://github.com/gardener/gardener/issues/4805)?

Reply:
Good point, I added it to the issue